sdk/eventhub/azure-eventhub/azure/eventhub/_pyamqp/aio/_transport_async.py

```diff
@@ -504,9 +504,9 @@ async def connect(self):
             raise ValueError(
                 "Please install aiohttp library to use websocket transport."
             )
-        except OSError:
+        except OSError as e:
             await self.session.close()
-            raise ConnectionError('Client Session Closed')
+            raise ConnectionError('Websocket connection closed: %r' % e) from e
```

Comment:
wondering if we need to do `% e` in the error message if we're already raising `from e`?